ProjectSource/src/test/java/com/kayako/sdk/utils/MessageLoader.java

```diff
@@ -0,0 +1,61 @@
+package com.kayako.sdk.utils;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public final class MessageLoader {
+
+  private static final String PATH =
+      "resources" + File.separator + "mockedMessages" + File.separator;
+  private static final String MSG_PREFIX = "message_";
+  private static final String TXT_EXTENSION = ".txt";
+  private static final String CONTENT_MARK = "*****";
+  private static final String TAB = "\t";
+
+  private MessageLoader() {
+    throw new AssertionError("This class cannot be instantiated");
+  }
+
+  public static String readFile(String msgFile, String contentFile, int contentLength){
+
+		String line;
+		StringBuilder messageBuilder = new StringBuilder();
+		int tabs;
```

Comment:
⚠️  The code uses both tabs and spaces for indentation. Being consistent with whitespace usage helps in displaying proper indentation regardless of IDE or editor. The recommended standard is using 2+ spaces instead of the tab character as spaces have the constant width.  

**[More details](https://confluence.devfactory.com/x/5RCUEw)**
          
## The possible solution in this case

Consider relying only on spaces and, for a consistent format, use the IDE formatting capabilities before committing the code.
 

<a id="3c8c9b60-50c4-11e8-b073-875130412ce4" href="none" codereview-uuid-reserved-tag />